requirements.txt

```diff
@@ -1,7 +1,10 @@
-numpy
-tensorflow
-sklearn
-gensim<=3.1
-h5py
+numpy>=1.13
+scipy==1.2.1
+tensorflow>=1.4
```

Comment:
I would say that here we need ~= operator

Reply:
Done.